### PR TITLE
Retry if the PR is unrebaseable

### PR DESCRIPTION
**Pull request by ddzero2c**
Trying to fix [issue 72](https://github.com/cirrus-actions/rebase/issues/72)

### CODE REVIEW

File: entrypoint.sh [hunk 1]
```diff
@@ -21,16 +21,36 @@ URI=https://api.github.com
...
+		sleep $RETRY_INTERVAL
+		continue
+	else
+		break
+	fi
+done
+
+if [[ "$REBASEABLE" != "true" || "$i" == "$MAX_RETRIES" ]] ; then
+	echo "GitHub doesn't think that the PR is rebaseable!"
+	echo "API response: $pr_resp"
```
Comment (fkorotkov) [inline entrypoint.sh:43]: Let's not print the response here. It was a CVE https://bugs.chromium.org/p/project-zero/issues/detail?id=2070

File: entrypoint.sh [hunk 2]
```diff
@@ -21,16 +21,36 @@ URI=https://api.github.com
...
+	if [[ "$REBASEABLE" == "null" ]]; then
+		echo "The PR is not ready to rebase, retry after $RETRY_INTERVAL seconds"
+		sleep $RETRY_INTERVAL
+		continue
+	else
+		break
+	fi
+done
+
+if [[ "$REBASEABLE" != "true" || "$i" == "$MAX_RETRIES" ]] ; then
```
Comment (fkorotkov) [inline entrypoint.sh:41]: Also no need to check for `i` value. Not even sure if it's still available here. 🤔 

```suggestion
if [[ "$REBASEABLE" != "true" ]] ; then
```

Comment (ddzero2c) [reply entrypoint.sh]: https://github.com/cirrus-actions/rebase/pull/75/commits/e4987e76cfabed2bbe500c3b855a435b58cf7947

File: entrypoint.sh (hunk 1 above)
Comment (ddzero2c) [reply entrypoint.sh]: https://github.com/cirrus-actions/rebase/pull/75/commits/e4987e76cfabed2bbe500c3b855a435b58cf7947

**Comment by fkorotkov** [comment]
@ddzero2c thank you for the PR!

**Pull request merged**